src/initialization.jl

```diff
@@ -100,7 +100,8 @@ Check if the algebraic constraints are satisfied, and error if they aren't. Retu
 the `u0` and `p` as-is, and is always successful if it returns. Valid only for
 `ODEProblem` and `DAEProblem`. Requires a `DEIntegrator` as its second argument.
 """
-function get_initial_values(prob::ODEProblem, integrator, f, alg::CheckInit,
+function get_initial_values(
+        prob::AbstractDEProblem, integrator::DEIntegrator, f, alg::CheckInit,
```

Comment:
This doesn't handle DAEProblem?

Reply:
There's an `AbstractDAEProblem` dispatch below

Reply:
But then this one should just be for ODEProblem?

Reply:
SDEProblem dispatches here too